src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_NUMBER = 100;
+    private static final Random random = new Random();
+    private static final ColorSupplier color = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        int value = random.nextInt(MAX_NUMBER);

```

Comment:
Please, pay attention to what you pass to this method and how it actually works. Now the maximum possible value, that this method can return, is 99.
![image](https://github.com/mate-academy/jv-lottery/assets/46851188/effb3435-1983-456c-bc0d-099bccb3ed9c)